content/faq/security/general.md

```diff
@@ -1,9 +1,88 @@
+
+Organizations set up in Docker use verified domains and any team member with an email domain other than what's verified is noted as a "Guest" in that org.
+
+### How long are Docker Hub logs available? 
+
+Docker provides various types of audit logs and log retention varies. For example, Hub Activity logs are available for 90 days. You are responsible for exporting logs or setting up drivers to their own internal systems.  
+
+### Can I export a list of all users with their assigned roles and privileges and if so, in what format?
+
+Using the [Export Members](../../docker-hub/members.md) feature, customers can export to CSV a list of their users with role and team information. 
+
+### How does Docker Desktop handle and store authentication information?
+
+Docker Desktop utilizes the host operating system's secure key management for handling and storing authentication tokens necessary for authenticating with image registries. On macOS, this is [Keychain](https://support.apple.com/guide/security/keychain-data-protection-secb0694df1a/web); on Windows, this is [Security and Identity API via Wincred](https://learn.microsoft.com/en-us/windows/win32/api/wincred/); and on Linux, this is [Pass](https://www.passwordstore.org/). 
+
+### How does Docker Hub secure passwords in storage and in transit? 
+
+This is applicable only when Docker Hub's application level password is used vs SSO/SAML. When SSO is used, Docker Hub does not store passwords. Application level passwords are hashed in storage (SHA-256) and encrypted in transit (TLS).
+
+### How do we de-provision access to CLI users who use personal access tokens instead of our IdP? We use SSO but not SCIM. 
+
+If SCIM is not enabled, you have to manually remove PAT users from the org in our system. When SCIM is used this is automated.
+
+### What metadata is collected from container images that Scout analyzes?
+
+For information about the metadata stored by Docker Scout, [Data handling](../../scout/data-handling.md).
+
+### To which portions of the host filesystem do containers have read and write access? Can containers running as root gain access to admin owned files or directories on the host? 
+
+File sharing (bind mount from the host filesystem) uses a user-space crafted file server (running in com.docker.backend as the user running docker desktop), so containers can’t gain any access that the user on the host doesn’t already have.
+
+### How are Extensions within the Marketplace vetting for security prior to placement? 
+
+Security vetting for extensions is on our roadmap however this vetting is not currently done. 
+
+At present in the marketplace, there are two types of extensions - reviewed and self-published. Reviewed extensions are used and reviewed by the Publisher team against a set of criteria, and if they pass they are included in the marketplace with a **Reviewed** label. Self-published extensions are automatically placed in the marketplace with a **Not reviewed** label. 
```

Comment:
I don't think we should be calling out the "Publisher team" here as its own entity.